api/server/httputils/httputils_write_json_go16.go

```diff
@@ -1,16 +0,0 @@
-// +build go1.6,!go1.7
```

Comment:
you could remove the build tag from here I guess https://github.com/moby/moby/blob/4f259698b07653e9e5220e097df79862f9e54b74/api/server/httputils/httputils_write_json.go#L1